UI/Dialogs/ConfigWizard.cs

```diff
@@ -397,6 +397,7 @@ private void button1_Click(object sender, EventArgs e)
             try {
                 if (!ValidateChildren())
                 {
+                    Log.Instance.log("The dialog cannot be closed. There are invalid values on some tab.", LogSeverity.Error);
```

Comment:
Does the dialog just silently not close in this case? Can we pop a dialog up to the user?

Reply:
Yeah, I figured, that's why I approved anyway. Thanks for the explanation.